specs/spec_helper.rb

```diff
@@ -0,0 +1,9 @@
+require 'simplecov'
+SimpleCov.start
+
+require 'minitest'
+require 'minitest/spec'
+require 'minitest/autorun'
+require 'minitest/reporters'
+
```

Comment:
If you require the far_mar module here, you won't need to require it or the individual data classes in each spec file. Let the module be in charge of assembling the classes it needs.